QuizApp/QuizApp/Extensions/UIImageView+Extensions.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+extension UIImageView {
```

Comment:
This is ok but for the next PR, investigate 3rd party libs that do this work for you and also cache the images ;)